packages/client/src/lib.js

```diff
@@ -13,11 +13,25 @@
  * ```
  * @module
  */
+
+// @ts-ignore module with no types
+import toIterable from 'stream-to-it'
+// @ts-ignore module with no types
+import { parallelMap } from 'streaming-iterables'
+import pRetry from 'p-retry'
+
+import { CarReader } from '@ipld/car'
+import { TreewalkCarSplitter } from 'carbites/treewalk'
+
 import * as API from './lib/interface.js'
 import * as Token from './token.js'
 import { fetch, File, Blob, FormData } from './platform.js'
 import { toGatewayURL } from './gateway.js'
 
+const MAX_STORE_RETRIES = 3
+const MAX_CONCURRENT_UPLOADS = 3
+const MAX_CHUNK_SIZE = 960 * 960 * 100 // chunk to ~96MB CARs
```

Comment:
Maybe it should be 
```suggestion
const MAX_CHUNK_SIZE = 1024 * 1024 * 96 // chunk to ~96MB CARs
```
because written code is chunking to 87.89063 MB

Reply:
thanks, but this was changes for testing purposes